packages/fxa-settings/src/models/pages/signup/key-stretch-experiment.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import {
+  IsOptional,
+  IsString,
+} from 'class-validator';
+import { bind, ModelDataProvider } from '../../../lib/model-data';
+
+export class KeyStretchExperiment extends ModelDataProvider {
```

Comment:
nit: Should this live in the `signup` folder? Or should we have an experiments one?

Reply:
Good call. I'll create an experiments folder.